index.html

```diff
@@ -640,6 +641,38 @@ <h2>Typography</h2>
 				<h2>Spinner</h2>
 				<ts-spinner data-visible data-size="medium" data-color="mono"> </ts-spinner>
 			</article>
+
+			<article>
+				<h2>Search</h2>
+				<div style="display: flex;">
+					<div style="width: 300px; margin: 5px;">
+						<ts-search placeholder="یا بخت یا اقبال" id="search" dir="rtl"></ts-search>
+					</div>
+					<div style="width: 600px; margin: 5px;">
+						<ts-search placeholder="Press Enter to search..." id="search2"></ts-search>
+					</div>
+				</div>
+				<div style="display: flex;">
+					<div style="width: 300px">
+						<div id="searchResult">No value</div>
+					</div>
+					<div style="width: 600px">
+						<div id="searchResult2">No value</div>
+					</div>
+				</div>
+				<button id="searchButton">Set focus on the right search</button>
+			</article>
+			<script>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The length of this JS script (10) exceeds the maximum set to 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3ALongJavaScriptCheck)